README.md

```diff
@@ -107,7 +120,7 @@ $ niet ".project.meta.name" /path/to/your/file.yaml
 my-project
 $ niet ".project.foo" /path/to/your/file.yaml
 bar
-$ niet ".project.list-items" /path/to/your/file.yaml
+$ niet ".project.list" /path/to/your/file.yaml
```

Comment:
could be great to have a different commit for changing list-items to list, it a bit annoying to review ^^

Reply:
No it sound like a bug so in a first time I prefer to simply transform these examples to search a little bit more in few days to eventually fix this behaviour...

Reply:
no, this is not a bug ^^ It's just an example in documentation .... ^^
But whatever, no problem

Reply:
I guess using a dash with jsmesg on a search introduce some errors... I need to check why jsmesg react like this and I guessing submit a fix to jmesg to avoid this behaviour.